.eslintignore

```diff
@@ -7,3 +7,4 @@
 /node_modules
 Gruntfile.js
 *.config.js
+writeConfig.js
```

Comment:
nit: Why do we need to add this to eslintignore? Is it just for the console line (and if so, why not just disable that check in the file)?